src/types/simple.rs

```diff
@@ -125,6 +122,8 @@ pub enum Container<T: PrimType> {
     Array(Box<T>, usize),
     /// Alias defined in AliasDefn or AliasDecl nodes.
     Alias(SmolStr),
+    /// An opaque operation that can be downcasted by the extensions that define it.
```

Comment:
```suggestion
    /// An opaque type that can be downcasted by the extensions that define it.
```

Reply:
Let's also add "this will always have a TypeTag contained within that of the Container" (so a `Container<SimpleType>` could have an Alias whose tag was Classic, say). Or words to that effect - CustomType's don't *have* a TypeTag yet until the next PR...

Reply:
The same concern does apply to Alias, of course! (But there we have only a name, which doesn't tell us much on its own, whereas here we have a CustomType)